tremor-pipeline/src/op/trickle/select.rs

```diff
@@ -137,7 +137,9 @@ impl std::default::Default for WindowImpl {
     fn default() -> Self {
         TumblingWindowOnTime {
             size: 15_000_000_000,
+            emit_empty_windows: false,
```

Comment:
I still think adding this is an extremely bad idea. We have been very careful not to give people the foot-gun of unbounded memory usage and had long discussions with users about why this is and quite a few tickets on `tremor has a memory leak` before we added this restriction to the grouping operator.

I think we shouldn't have it at all as it increases the sufrace and by that complexity for users and no one has ever asked for this. But if we have to have it for some reason this should be bounded the same way as the grouping operator is into something like `max_empty_windows`

Reply:
I am open to both. But i think this means @Licenser and @darach you need to go into the thunderdome and decide how we want this.

Reply:
Your opinion is as valuable as Darachs or mine :D, sorry you can't not have one.

Reply:
I think it is ok to have this config when the default is to not emit empty windows.

Reply:
that makes 2 vs 1 :P leave it in then, but can we at least instead of making it unbounded bound it the same way we bound other parts growth parts?

Reply:
i would very much like to avoid to add another config just because the first one is not good enough. The `max_empty_windows` is really too much. Then i would prefer to leave it out completely, honestly.